php/class-fm-bylines-cli.php

```diff
@@ -0,0 +1,134 @@
+<?php
+WP_CLI::add_command( 'fm-bylines', 'FM_Bylines_CLI' );
+
+class FM_Bylines_CLI extends WP_CLI_Command {
+
+	/**
+	 * Migrate Co-Authors Plus Guest Authors and linked accounts to FM Bylines
+	 *
+	 */
+	public function migrate_coauthors() {
+		global $coauthors_plus;
+
+		$cap_terms = get_terms( $coauthors_plus->coauthor_taxonomy, array( 'hide_empty' => false ) );
+
+		WP_CLI::line( 'Migrating Co-Authors Plus guest-authors to bylines:' );
+
+		foreach ( $cap_terms as $cap_term ) {
+			$args = array(
+				'post_type' => $coauthors_plus->guest_authors->post_type,
+				'name' => $cap_term->slug,
+				'post_status' => 'publish',
+				'numberposts' => 1,
+			);
+			$cap_post = get_posts( $args );
+			if ( ! empty( $cap_post[0]->ID ) ) {
+				$cap_meta = apply_filters( 'fm_bylines_cli_cap_meta', get_post_meta( $cap_post[0]->ID ), $cap_post[0], $cap_term );
```

Comment:
nope, this hook is just here so you can do something with the meta data before you do the migration. So if you want to clean it or check for values etc.